### DISCUSSION TITLE
Ability to remember Workflow selection choice

### DISCUSSION THREAD

**Discussion by seanmiller26** [zooniverse/front-end-monorepo#6211] [Feature Requests]
From this Talk thread: https://www.zooniverse.org/projects/cobalt-lensing/black-hole-hunters/talk/4564/3371395

Should we add a small checkbox which would "Remember my selection"? This could be especially useful in projects like Black Hole Hunters which has a Training Workflow. Once you complete the training, you wouldn't need to do that Workflow again.

![Image](https://github.com/zooniverse/front-end-monorepo/assets/89043489/32fbfa7a-319c-435c-bd77-311961fcdf55)

**Comment by eatyourgreens** [comment]
I’ve opened #6216 to remember your workflow selection in a browser session cookie. 

Here’s a short video of how that looks on a project with two workflows. 
https://github.com/zooniverse/front-end-monorepo/pull/6216#issuecomment-2295168669
